src/client/src/containers/DependencyInfo/styles.module.css

```diff
@@ -17,30 +17,42 @@
   text-decoration: none;
 }
 
-.borderGreen {
-  border: 1px solid #99ffa7;
+.body {
+  margin-left: 6px;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 12px;
+  /* or 222% */
```

Comment:
What is this comment here for, just wondering

Reply:
oops, i think this was copied over from Christina's Figma